docs/source/DeveloperGuide/ReleaseProcedure/index.rst

```diff
@@ -26,237 +29,42 @@ The release manager needs to:
-One typical conflict is about the content of the ``release.notes`` file.
-
-From now on, the process will look very similar to what we have already done for
-creating tag v7r1p37. We should then repeat the process for v7r2::
-
-  $ vim release.notes
-  $ vim __init__.py
-
-Merge PRs (if any), then save the files above. Then::
-
-  $ git commit -a  # this will commit the changes we made to the release notes in rel-v7r2 local branch
-  $ git fetch release  # this will bring in the updated release/rel-v7r2 branch from the github repository
-  $ git rebase --no-ff release/rel-v7r2  # this will rebase the current rel-v7r2 branch to the content of release/rel-v7r2
-  $ git push release rel-v7r2  # we push to the *release* remote the tag just created, and the rel-v7r2 branch.
-
-Now, check GA and if everything is fine::
-
-  $ git tag v7r2p8  # this will create a tag, from the current branch, in the local repository
-  $ git push v7r2p8  # we push to the *release* remote the tag just created
-
-The *master* branch of DIRAC always contains the latest stable release.
-If this corresponds to rel-v7r2, we should make sure that this is updated::
-
-  $ git push release rel-v7r2:master
-
-Repeat the process for every "upper" release branch.
-When the release branch of the latest stable version is changed, i.e. from rel-v7r2 to rel-v7r3, the URL of the CI status badge in the README should be edited.
-
-The *integration* branch is also receiving new features to go into the next release.
-The *integration* branch also contains the ``releases.cfg`` file, which holds all the versions of DIRAC
-together with the dependencies among the different packages. 
-
-From the *integration* branch we also do all the tags of *pre-release* versions, that can be then installed
-with standard tools on test DIRAC servers. 
-
-The procedure for creating pre-releases is very similar to creating releases::
-
-  $ vim release.notes
-  $ vim __init__.py
-  $ vim releases.cfg  # add the created tags (all of them, releases and pre-releases)
-
-Merge all the PRs targeting integration that have been approved (if any), then save the files above. Then::
-
-  $ git commit -a
-  $ git fetch release
-  $ git rebase --no-ff release/integration
-  $ git push release integration
-
-Wait for tests on GA to complete and then::
-
-  $ git tag v7r3-pre9
-  $ git push v7r3-pre9
-
-
-2. Making basic verifications
-=============================
-
-All unit and integration tests are automatically run by `GitHub Actions <https://github.com/DIRACGrid/DIRAC/actions>`_
-
-GitHub actions also runs on all the Pull Requests, so if for all the PRs merged GitHub Actions didn't show any problem,
-there's a good chance (but NOT the certainty) that the created tags are also sane.
-
-From version v7r2, python3 releases are automatically created (again, by GitHub Actions)
-when a tag is pushed, and should be found on `pypi <https://pypi.org/project/DIRAC/>`_.
-
-
-3. Deploying python2 DIRAC tarballs
```

Comment:
Well, you have removed this part about python2 releases, but I would at least keep referencing it since v7r* releases will keep existing for a bit.

Reply:
I thought it would be cleaner to leave the v7r3 branch as the canonical source of the legacy release procedure?

Reply:
It's not a big deal really, but worth putting at least an explicit link at the top of the page for v7r* releases. The v7r* tags will still needs to be created by hand, as well as the updates of releases.notes and releases.cfg and __init__.py.